packages/react-native-executorch/common/rnexecutorch/host_objects/JsiConversions.h

```diff
@@ -19,6 +23,11 @@ inline double getValue<double>(const jsi::Value &val, jsi::Runtime &runtime) {
   return val.asNumber();
 }
 
+template <>
```

Comment:
Consider replacing the specialization for int and double with 
```
template<typename T>
concept IsNumeric = std::is_arithmetic<T>::value;

template <typename T>
requires IsNumeric<T>
inline T getValue(const jsi::Value &val, jsi::Runtime &runtime) {
    static_assert(std::is_integral<T>::value || std::is_floating_point<T>::value,
                  "Only integral and floating-point types are supported");
    return static_cast<T>(val.asNumber());
}
```
This will enable for all floating and numeric which I don't know if it's desired, so feel free to skip it.